v19.1/architecture/replication-layer.md

```diff
@@ -83,7 +83,7 @@ Because CockroachDB serves reads from a range's leaseholder, it benefits your cl
 
 {{site.data.alerts.callout_info}}
 
-This feature is also called [Follow-the-Workload](http://localhost:4000/docs/stable/demo-follow-the-workload.html) in our documentation.
+This feature is also called [Follow-the-Workload](https://www.cockroachlabs.com/docs/stable/demo-follow-the-workload.html) in our documentation.
```

Comment:
This link should be relative, i.e.

```
[Follow the Workload](../demo-follow-the-workload.html)
```

That way it always refers to the same version as this page.

Reply:
Done.